.github/workflows/test_deployment.yml

```diff
@@ -25,4 +25,4 @@ jobs:
     - name: Build the Docker image
       run: docker-compose up -d
     - name: Creating new release package
-      run: docker-compose exec -T snowshu python3 setup.py sdist bdist_wheel
```

Comment:
can't see any difference besides delete or added new line

Reply:
Ah, this change was also included in #103 which ended up getting merged before this PR. Missed that fact, going to close this one out.